03/app.js

```diff
@@ -1,5 +1,25 @@
-document.addEventListener('DOMContentLoaded', init);
+document.addEventListener("DOMContentLoaded", init);
 
 function init() {
-    console.log('DOM');
+  console.log("DOM");
+  
+  const button = document.querySelector("button");
+  const span = document.querySelector("span");
+
+  fetchIp().then((resp) => {
+    button.addEventListener("click", () => showIp(resp, span));
+  });
```

Comment:
Powinno być odwrotnie. Na klik uruchamiamy zapytanie do API i wyświetlamy adres IP.
Tutaj kolejność może mieć znaczenie. Jeśli między kliknięciem w button, a załadowaniem strony zmienimy sieć to adres IP będzie nieprawidłowy :)